chore(perf): Remove references to legacy APDEX, USER_MISERY descriptions

shruthilayaj opened the pull request:
Remove references to legacy apdex.

shruthilayaj committed:
chore(perf): Remove references to legacy APDEX, USER_MISERY descriptions

narsaynorath reviewed (approved):
Looks good to me. From what I understand this will just pull the definitions for `APDEX_NEW` and `USER_MISERY_NEW` and these definitions don't reference any organization level values

scttcper committed:
feat(workflow): Remove issue-percent-filters flag (#32011)

markstory committed:
chore(ui) Add role to improve selectors in tests (#32119)

Add a role so we can select the ring as an image without having to use
`container.querySelector`

Refs getsentry/getsentry#7087

scttcper committed:
feat(alertStatus): Add pagination to issues table (#32019)

RyanSkonnord committed:
ref(access): Use OrganizationGlobalAccess in more cases (#32020)

Use the OrganizationGlobalAccess subclass (replacing concrete Access
objects) to represent where superusers and Sentry apps have access to
all an organization's teams and projects. Consistently expose those sets
of teams and projects as attributes (previously done for Sentry apps,
but not the superuser or global auth case).

Filter the `teams` attribute by visibility for all Access objects, for
consistency with OrganizationMember.get_teams (used by from_member).

shruthilayaj committed:
chore(perf): Rename apdex_new, user_misery_new (#32126)

RyanSkonnord committed:
fix(auth): Remove Okta migration workaround (#31800)

Delete "organizations:sso-migration" feature flag and associated logic.

shruthilayaj committed:
Merge branch 'master' into chore/remove-references-to-old-apdex

shruthilayaj committed:
Merge branch 'master' into chore/remove-references-to-old-apdex

shruthilayaj committed:
empty commit

shruthilayaj committed:
Merge branch 'master' into chore/remove-references-to-old-apdex
